docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,99 @@
+- [PMM-12310](https://jira.percona.com/browse/PMM-12310) - Links from PMM to documentation may change, causing "broken links" in older PMM versions due to document structure changes. We have replaced all links with Short.io links specific to each document to address this. This way, we can maintain the PMM-to-doc links using a URL shortener. This ensures that the links remain accessible even if the document structure changes.
+- [PMM-12457](https://jira.percona.com/browse/PMM-12457) - We have updated the **PMM agent down** alert template in PMM with a new `node_name` property. This allows users to easily identify the node where the failure occurred.
+- [PMM-12488](https://jira.percona.com/browse/PMM-12488) - VictoriaMetrics has been updated to v1.93.4.
+- [PMM-12500](https://jira.percona.com/browse/PMM-12500) - [Tech Preview] K8s monitoring for Percona Operator users - basic dashboard
+
+
+## Bugs Fixed
+
+- [PMM-10145](https://jira.percona.com/browse/PMM-10145) - When we installed an AMI image in AWS and added an Amazon RDS instance in the creating state, it caused errors. The issue has been resolved now.
+- [PMM-12173](https://jira.percona.com/browse/PMM-12173) - On adding several clients to PMM, unexpected and unauthorized errors caused PMM to stop responding. The issue has since been resolved now.
+- [PMM-12221](https://jira.percona.com/browse/PMM-12221) - Fixed an issue where the user was logged out during an upgrade.
+- [PMM-12344](https://jira.percona.com/browse/PMM-12344) - An error was displayed on the **Explain** tab after selecting a PostgreSQL query. The issue has been resolved now.
+- [PMM-12361](https://jira.percona.com/browse/PMM-12361) - The command `pmm-admin inventory` returned GO errors. The issue has been reolved now.
+- [PMM-12382](https://jira.percona.com/browse/PMM-12382) - Fixed an issue where the upper-case custom labels were not being accepted.
+- [PMM-12420](https://jira.percona.com/browse/PMM-12420) - PMM upgrade was failing due to migration from SQLite to PGSQL. The issue has been resolved now.
+- [PMM-11371](https://jira.percona.com/browse/PMM-11371) - pmm-admin config failed to register agent to pmm-server. The issue has been reolved now.
+- [PMM-11603](https://jira.percona.com/browse/PMM-11603) - When running `pmm-agent` outside of systemd, adding an agent causes errors. The issue has been resolved now.
+- [PMM-11651](https://jira.percona.com/browse/PMM-11651) - When a user has both full access and limited access roles, only the limited role's metrics are displayed. The issue has been resolved now.
+- [PMM-12146](https://jira.percona.com/browse/PMM-12146) - Dead Tuples graph on PostgreSQL Vacuum Monitoring Dashboard displayed invalid percentage. The 
+- [PMM-12448](https://jira.percona.com/browse/PMM-12448) - Can't start backup if artifact with empty service_id exists.
```

Comment:
Let's maybe tell what has been done here )

Reply:
This will be worked upon by @catalina-adam

Reply:
Done